.github/workflows/spell_check.yml

```diff
@@ -26,5 +26,11 @@ jobs:
 
       - name: Run cspell
         run: |-
-          cspell lint "**"
-          echo -e "\e[91mNote: If the detected words are expected, add them into ".cspell.json" which is at the repo root folder.\e[0m" 
+          cspell lint "**" > cspell_check.log || true
```

Comment:
will it always create a new file? if so, have we added the log file to gitignore?

Reply:
Yes it always create this file and yes it's already in `.gitignore`